docs/core/install/linux-ubuntu.md

```diff
@@ -0,0 +1,364 @@
+---
+title: Install .NET Core on Ubuntu - .NET Core
+description: Demonstrates the various ways to install .NET Core SDK and .NET Core Runtime on Ubuntu.
+author: thraka
+ms.author: adegeo
+ms.date: 05/18/2020
+---
+
+# Install .NET Core SDK and .NET Core Runtime on Ubuntu
+
+.NET Core is supported on different Ubuntu versions. This article describes how to install .NET Core on Ubuntu with APT. When an Ubuntu version falls out of support, .NET Core is no longer supported with that version. However, these instructions may help you to get .NET Core running on those versions, even though it isn't supported.
+
+## Supported distributions
+
+The following is a list of currently supported .NET Core releases and the versions of Ubuntu they're supported on. These versions remain supported until either the version of [.NET Core reaches end-of-support](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) or the version of [Ubuntu reaches end-of-life](https://wiki.ubuntu.com/Releases).
+
+For the best compatibility, choose a long-term release (LTS) version.
+
+| .NET Core | Ubuntu version (LTS) | Ubuntu version |
+|-----------|----------------------|----------------|
+| 2.1 (LTS) | 16.04, 18.04, 20.04  | 19.10          |
+| 3.1 (LTS) | 16.04, 18.04, 20.04  | 19.10          |
+
+.NET Core versions no longer supported and the Ubuntu LTS releases they were supported with:
+
+| .NET Core | Ubuntu version (LTS) |
+|-----------|----------------------|
+| 3.0       | 16.04, 18.04         |
+| 2.2       | 16.04, 18.04         |
+| 2.0       | 14.04, 16.04, 18.04  |
+
+## APT 20.04 (supported)
+
+Before installing .NET, you'll need to:
+
+- Add the Microsoft package signing key to the list of trusted keys.
+- Add the repository to the package manager.
+- Install required dependencies.
+
+> [!TIP]
+> Registering the key only needs to be done once per machine.
+
+Open a terminal and run the following commands.
+
+```bash
+wget https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
+sudo dpkg -i packages-microsoft-prod.deb
+```
+
+[!INCLUDE [linux-apt-install-31](includes/linux-apt-install-31.md)]
+
+## APT 19.10 (supported)
```

Comment:
It's a little alarming how large the page got from this, with repeating text... and it will only grow larger as more Ubuntu versions come out. But it is the most reliable way to do this but keep each section infinitely customizable (e.g. for source-build vs. Msft, or distro-specific workarounds if we miss dotnet-runtime-deps for an extended period again).